### PR TITLE
OSDOCS-1161 Add log-send-hostname parameter

### PR DESCRIPTION
**Pull request by bmcelvee**
https://issues.redhat.com/browse/OSDOCS-1161

### CODE REVIEW

**Comment by bmcelvee** [comment]
@quarterpin would you mind reviewing this PR, please? thanks!

File: modules/nw-route-specific-annotations.adoc [hunk 1]
```diff
@@ -29,6 +29,7 @@ and "-". The default is the hashed internal key name for the route. |
 |`router.openshift.io/haproxy.health.check.interval`| Sets the interval for the back-end health checks. (TimeUnits) | `ROUTER_BACKEND_CHECK_INTERVAL`
 |`haproxy.router.openshift.io/ip_whitelist` | Sets a whitelist for the route. |
 |`haproxy.router.openshift.io/hsts_header` | Sets a Strict-Transport-Security header for the edge terminated or re-encrypt route. |
+|`haproxy.router.openshift.io/log-send-hostname` | Sets the hostname field in the Syslog header. Uses the hostname of the system.
```
Comment (quarterpin) [inline modules/nw-route-specific-annotations.adoc:32]: In addition, we can also mention that  "this gets applied/enabled by default if any ingress API logging method (sidecar or Syslog facility) is enabled for the router"

Other than that, lgtm.

Comment (bmcelvee) [reply modules/nw-route-specific-annotations.adoc]: Thank you! Added.

**Comment by bmcelvee** [comment]
@openshift/team-documentation please peer review, thanks!

File: modules/nw-route-specific-annotations.adoc [hunk 2]
```diff
@@ -29,6 +29,7 @@ and "-". The default is the hashed internal key name for the route. |
 |`router.openshift.io/haproxy.health.check.interval`| Sets the interval for the back-end health checks. (TimeUnits) | `ROUTER_BACKEND_CHECK_INTERVAL`
 |`haproxy.router.openshift.io/ip_whitelist` | Sets a whitelist for the route. |
 |`haproxy.router.openshift.io/hsts_header` | Sets a Strict-Transport-Security header for the edge terminated or re-encrypt route. |
+|`haproxy.router.openshift.io/log-send-hostname` | Sets the hostname field in the Syslog header. Uses the hostname of the system. `log-send-hostname` is enabled by default if any Ingress API logging method, such as sidecar or Syslog facility, is enabled for the router. |
```
Comment (bergerhoffer) [inline modules/nw-route-specific-annotations.adoc:32]: * The first "hostname" should be in backticks if it's an actual field.
* The second "hostname" instance should be two words.

**Review by bergerhoffer — COMMENTED** [review]
Just one thing!

File: modules/nw-route-specific-annotations.adoc (hunk 2 above)
Comment (bmcelvee) [reply modules/nw-route-specific-annotations.adoc:32]: Thank you!

**Pull request merged**